api/src/controllers/record-controller.js

```diff
@@ -212,7 +197,7 @@ exports.protectedUnPublish = function (args, res, next) {
  * @param {*} next
  * @returns
  */
-exports.publicHead = function (args, res, next) {
+exports.publicHead = function(args, res, next) {
```

Comment:
We should get rid of these since we don't support publicHead.  Also - keep whitespace changes for other commits unless you're fixing logic in said function to keep reviews to a minimum.

Reply:
Removed.